docs/config.md

```diff
@@ -64,9 +64,11 @@ Supported keys include:
 | `artifacts.pipelinerun.format` | The format to store `PipelineRun` payloads in. | `in-toto`, `slsa/v1`| `in-toto` |
 | `artifacts.pipelinerun.storage` | The storage backend to store `PipelineRun` signatures in. Multiple backends can be specified with comma-separated list ("tekton,oci"). To disable the `PipelineRun` artifact input an empty string ("").  | `tekton`, `oci`, `gcs`, `docdb`, `grafeas` | `tekton` |
 | `artifacts.pipelinerun.signer` | The signature backend to sign `PipelineRun` payloads with. | `x509`, `kms` | `x509` |
+| `artifacts.pipelinerun.enable-deep-inspection` | This boolean option will configure whether Chains should inspect child taskruns for captureing inputs/outputs within a pipelinerun. `false` means that Chains only checks pipeline level results, whereas `true` means Chains inspects both pipeline level and task level results. | `true`, `false` | `false` | 
```

Comment:
Although it's a boolean, it's actually a string in the ConfigMap, right? Maybe we can say `"false"` and `"true"` instead of `false` and `true`?